h2d/Mask.hx

```diff
@@ -95,7 +159,7 @@ class Mask extends Object {
 				c.posChanged = true;
 			posChanged = false;
 		}
-		addBounds(relativeTo, out, 0, 0, width, height);
+		addBounds(relativeTo, out, scrollX, scrollY, width, height);

```

Comment:
This does not seem correct.

Reply:
Why? With that, Mask will report proper content bounds. Sample below showcases it. Green outline is mask boundaries, red is reported boundaries.
Currently: 
![image](https://user-images.githubusercontent.com/1061373/80307427-b0a8f980-87d1-11ea-8268-fc12d2072bdd.png)
With `-scrollX, -scrollY` it does not clip bounds properly and actually offsets them:
![image](https://user-images.githubusercontent.com/1061373/80307468-e51cb580-87d1-11ea-8432-c7e0b101a09c.png)
And with `0, 0` it just reports offset boundaries of Mask:
![image](https://user-images.githubusercontent.com/1061373/80307483-01205700-87d2-11ea-9283-5ac28d7ae41e.png)
So everything is correct. :)